lib/experimental/fonts-library/class-wp-fonts-library.php

```diff
@@ -0,0 +1,618 @@
+<?php
+include(ABSPATH . "wp-admin/includes/admin.php");
+
+/**
+ * Font library class.
+ *
+ * @package gutenberg
+ * @since x.x.x
+ * 
+ */
+class WP_Fonts_Library extends WP_REST_Controller {
+    public function __construct() {
+        $this->wp_fonts_dir = path_join( WP_CONTENT_DIR, 'fonts' );
+        $this->relative_fonts_path = content_url('/fonts/');
+
+		$this->rest_base        = 'fonts_library';
+		$this->namespace        = 'wp/v2';
+
+        $this->post_type        = 'wp_fonts_library';
+        $this->post_name        = 'wp_fonts_library';
+        $this->post_title       = 'Fonts Library'; // Do not translate
+        $this->post_status      = 'publish';
+
+        $this->base_post_query  = array  (
+            'post_type'      => $this->post_type,
+            'post_title'     => $this->post_title,
+            'post_status'    => $this->post_status,
+            'post_name'      => $this->post_name,
+            'name'           => $this->post_name,
+            'posts_per_page' => 1,
+        );
+	}
+
+	const ALLOWED_FONT_MIME_TYPES = array(
+		'otf'   => 'font/otf',
+		'ttf'   => 'font/ttf',
+		'woff'  => 'font/woff',
+		'woff2' => 'font/woff2',
+	);
+
+    function register_routes () {
```

Comment:
```suggestion
    public function register_routes () {
```
Let's explicitly declare the visibility for methods in this class.
[Visibility should always be declared](https://developer.wordpress.org/coding-standards/wordpress-coding-standards/php/#visibility-should-always-be-declared).

As I understand it, the methods in this class should be either public or protected unless you want to forbid extending your methods in the descendant classes. Then they should be specified as `private`.
Since `register_routes()` is inherited from the parent class, it should have the same visibility, i.e., public.